### PR TITLE
Canvas LMS

### PR DESCRIPTION
**Pull request by woto**
Thank you for taking the time to work on a PR for Awesome-Selfhosted!

To ensure your PR is dealt with swiftly please check the following:

- [x] Your submissions are formatted according to the guidelines. 
        
    ``[Name](http://homepage/) - Short description. ([Demo](http://url.to/demo), [Source Code](http://url.of/source/code)) `License` `Language```
- [x] Your additions are ordered alphabetically.
- [x] Your additions are [free software](https://en.wikipedia.org/wiki/Free_software), or if not they have been added to [non-free](non-free.md).
- [x] Any licenses you have added are in our [list of licenses](https://github.com/Kickball/awesome-selfhosted/blob/master/README.md#list-of-licenses).
- [x] You have searched the repository for any relevant issues or PRs.
- [x] Any category you are creating has the minimum requirement of 3 items.

### CODE REVIEW

**Comment by danger-awesome-Selfhosted** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Error">
          1 Error
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:no_entry_sign:</td>
      <td data-sticky="false">Found 1 entries with syntax error(s).</td>
    </tr>
  </tbody>
</table>
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">Please check PR guidelines and check the boxes.</td>
    </tr>
  </tbody>
</table>

#### Syntax Issues

 Name | Entry
----|----------------------
Canvas LMS |   * [Canvas LMS](https://github.com/instructure/canvas-lms) - Canvas is the trusted, open-source learning management system (LMS) that's revolutionizing the way we educate `Ruby`


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

File: README.md [hunk 1]
```diff
@@ -682,7 +682,8 @@ See https://github.com/n1trux/awesome-sysadmin#distributed-filesystems
 ## Learning and Courses
 
 **[`^        back to top        ^`](#)**
-
+  
+  * [Canvas LMS](https://github.com/instructure/canvas-lms) - Canvas is the trusted, open-source learning management system (LMS) that's revolutionizing the way we educate `Ruby`
```
Comment (nodiscc) [inline README.md]: missing final `.`

**Comment by woto** [comment]
Im too lazy to fix it )
May be later

4 марта 2017 г. 4:38 PM пользователь "nodiscc" <notifications@github.com>
написал:

**Comment by Kickball** [comment]
@nodiscc I've updated the entry, the formatting should be sorted now. Please take a look at the demo, it is the live cloud version of the software, that is fine right?

**Comment by nodiscc** [comment]
Yep

**Pull request merged**